src/Hydrators/IngredientEntityHydrator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
```

Comment:
needs to include the namespace at the top of the script for autoloading.

`namespace Mamoi\Entities;`

Check the IngredientEntity class for reference